openshift/odo
# Modifies odo app commands to work outside a directory

fixes #1646 (only the app part) 

This PR makes the `app` commands work outside a odo component directory if a project flag is given or the current project is used.

How to test:
Create a odo component and push. Then switch to a different directory containing no odo component. Then run the below commands

1.  `odo app list`
2.  `odo app delete <app-name>`
3.  `odo app describe <app-name>`

These commands should work outside a odo component directory
